frontend/packages/network-attachment-definition-plugin/src/models/index.ts

```diff
@@ -31,10 +31,10 @@ export const HyperConvergedModel: K8sKind = {
   labelPlural: 'HyperConverged Clusters',
   apiVersion: 'v1alpha1',
   apiGroup: 'hco.kubevirt.io',
-  plural: 'hyperconverged',
-  namespaced: false,
-  abbr: 'SRNNPM', // TODO check on this
-  kind: 'hyperconverged',
+  plural: 'hyperconvergeds',
+  namespaced: true,
+  abbr: 'hco',
```

Comment:
this should be in uppercase
```suggestion
  abbr: 'HCO',
```

Reply:
I believe this is the abbreviation for `kubectl`/`oc`, so you can call `oc get hco` instead of `oc get hyperconverged`. It should remain lower-case, following the CRD https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/deploy/crds/hco.crd.yaml#L18. Please correct me if I'm wrong and this is some UI abbreviation.

Reply:
these are UI abbreviations

Reply:
Fixed!